src/components/pipelines-overview/PipelineRunsDurationCard.tsx

```diff
@@ -40,41 +71,41 @@ const PipelinesRunsDurationCard: React.FC<PipelinesRunsDurationProps> = ({
         <Divider />
         <CardBody>
           <Grid hasGutter className="pipeline-overview__duration-card__grid">
-            <GridItem span={8}>
+            <GridItem span={7}>
               <span>
                 <MonitoringIcon className="pipeline-overview__duration-card__icon" />
                 {t('Average Duration')}
               </span>
             </GridItem>
-            <GridItem span={4}>
+            <GridItem span={5}>
               <span className="pipeline-overview__duration-card__value">
-                {summaryData['avg-duration']}
+                {summaryData['avg_duration']}
```

Comment:
If `summaryData['avg_duration']` is null, then add '-' in UI. @vikram-raj is this fine? to show '-' for ns where data is not there.

Reply:
Agree to add `- ` until data gets fetched.

Reply:
Done